.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_SUPABASE_URL=https://skmhrplxykgellccevso.supabase.co
```

Comment:
Please don't commit those keys, If someone knows it, they can change our DB

Reply:
Noted it

Reply:
Reverted
https://github.com/cam-drei/kids-study-front-end/pull/1/commits/1b42e3a417af0fe38d1d8bc5f47a89f24a294bda